setup.py

```diff
@@ -73,11 +108,15 @@ def run(self):
         super().run()
 
 
+# this option is only valid in setup.py
```

Comment:
Why do we need to exclude this argument?

Reply:
I think it was in previous for making ci test passed. (because of sdist problem was existing on that time.)
Lemme try to revert it.